packages/cli/generators/app/templates/src/application.ts.ejs

```diff
@@ -18,14 +18,17 @@ export class <%= project.applicationName %> extends BootMixin(RestApplication) {
         dirs: ['controllers'],
         extensions: ['.controller.js'],
         nested: true,
-      }
-    }
+      },
+    };
   }
 
   async start() {
+    const server = await this.getServer(RestServer);
+    // Set up the custom sequence
+    server.sequence(MySequence);
```

Comment:
Configuring the application inside `app.start()` is an antipattern we are trying to get rid of, see https://github.com/strongloop/loopback-next/pull/968 and https://github.com/strongloop/loopback-next/pull/994.

I'll send a PR to fix this.

Reply:
See https://github.com/strongloop/loopback-next/pull/1051